config/terraform.spc

```diff
@@ -2,8 +2,14 @@ connection "terraform" {
   plugin = "terraform"
 
   # Paths is a list of locations to search for Terraform configuration files.
-  # Wildcards are supported per https://golang.org/pkg/path/filepath/#Match
-  # Exact file paths can have any name. Wildcard based matches must have an
-  # extension of .tf (case insensitive).
-  # paths = [ "/path/to/dir/*", "/path/to/exact/custom.tf" ]
+  # Wildcard based searches are supported.
+  # For example:
+  #  - "*" matches all files in a directory.
+  #  - "**" matches all files in a directory, and all the sub-directories in it.
+  #  - "./*" matches all files in current working directory.
+  #  - "../*" matches all files in parent of current working directory.
+  #  - "steampipe*" matches all files starting with "steampipe".
+  # Exact file paths can have any name, i.e. `"/path/to/exact/custom.tf"`.
+  # Default set to current working directory.
+  paths = [ "./*.tf" ]
```

Comment:
```suggestion
  paths = [ "*.tf" ]
```

This change keeps the default functionally the same since we support relative paths, but looks a bit cleaner